infra/modules/providers/azure/service-plan/variables.tf

```diff
@@ -19,18 +20,18 @@ variable "service_plan_name" {
 }
 
 variable "service_plan_tier" {
-    description = "The tier under which the service plan is created. Details can be found at https://docs.microsoft.com/en-us/azure/app-service/overview-hosting-plans"
-    default     = "Isolated"
+  description = "The tier under which the service plan is created. Details can be found at https://docs.microsoft.com/en-us/azure/app-service/overview-hosting-plans"
+  default     = "Isolated"
```

Comment:
This should not default to isolated as some developers will not need to run this on a private vnet. If a developer needs to run this in a vnet then we should provide a template to automate that and add this default in the relevant TF template(s).

Reply:
This is what we had discussed in the meeting on call with @code4clouds. I did not have a default value before but we agreed to add this default for Service plans inside vnet based on the Game plan document.

Reply:
we discussed adding this default in the template that stands up a vnet, not the module. Most users of this module will not need to run this in isolated mode.

Reply:
This is a design decision that should be approved by @James-Nance @day-jeff .  If we are building an Enterprise level App Reference then Isolated should be mandatory as this is the most secure way to protect the information flowing between the services.  Enterprise App Reference in my head = PII, information that shouldn't be tampered or exposed outside of a VNET.

Reply:
Generally speaking, we still  need to do more research to double check whether isolated app service is required for private vnets / app services. This particular  module will be used by developers for app service plans which don’t necessarily require a private vnet. Isolated app services may make sense in certain scenarios but are  expensive [Instance](https://azure.microsoft.com/en-us/pricing/calculator/?service=app-service#app-service5dfb67a9-b49c-4c2c-82c4-9b726388f203). This decision is specific to the deployment environment(template) as this module will be used for  azure customer / developer(s) interested in deploying app services. In summary, private vnet is out of scope for this particular user story.